spyderlib/widgets/browser.py

```diff
@@ -10,10 +10,12 @@
 import sys
 
 # Third party imports
+from qtpy import PYQT5

```

Comment:
You're right. I was using it before I noticed we need a more fine grained constant (`WEBENGINE`) to activate/deactivate some things according to WebEngine or WebKit.